src/components/TodoForm.js

```diff
@@ -0,0 +1,20 @@
+import Reactת, {useState} from 'react'
+
+export const TodoForm = ({addTodo}) => {
+    const [value, setValue]=useState("")
+    const handleSubmit= e =>{
+        e.preventDefault();
+
+       addTodo(value);
+
+       setValue("")
+    }
+    return (
+     <form className = 'TodoForm' onSubmit={handleSubmit}>
```

Comment:
you dont have to use form on any button, you can use the onclick of a button in most cased, usually if you prevent the defualt functionallity of a onsubmit event, you dont really need the form